### PR TITLE
Relicense from AGPLv3+ to GPLv3

### PR DESCRIPTION
**Pull request by mwhudson**
Update LICENSE, headers and metadata in debian/copyright and setup.py

### CODE REVIEW

File: LICENSE
Comment (dbungert) [inline LICENSE:1]: https://www.gnu.org/licenses/gpl-3.0.txt differs slightly from here, mostly with url updates.  The old urls are still functional.

File: bin/probert [hunk 1]
```diff
@@ -2,16 +2,15 @@
 # Copyright 2015 Canonical, Ltd.
 #
 # This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero General Public License as
-# published by the Free Software Foundation, either version 3 of the
-# License, or (at your option) any later version.
+# it under the terms of the GNU General Public License as
+# the Free Software Foundation, version 3.
```
Comment (dbungert) [inline bin/probert:6]:
```suggestion
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, version 3.
```

Comment (mwhudson) [reply bin/probert]: fixed

File: LICENSE
Comment (mwhudson) [reply LICENSE:1]: updated

**Review by dbungert — APPROVED** [review]

**Pull request merged**